### PR TITLE
Added pop window css

### PR DESCRIPTION
**Pull request by Pazzo97**
This PR is about pop up window on portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -199,9 +199,68 @@ <h2>Multi-Post Stories</h2>
...
+              </div>
+
+              <div class="modal-picture"><img src="images/SnapshootPortfolio.png" alt="proect picture"></div>
+
+              <div class="modal-body">
+                <p class="mobile-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+                  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+                  scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting ever
+                  since the 1500s, when an unknown printer took a galley of type veris lapoa todoe. </p>
+
```
Comment (micheaol) [inline index.html]: Please ensure that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). You still have some hard-coded data in your html file. Kindly remove them

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Pazzo97 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @Pazzo97

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comments_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -17,3 +17,154 @@ openMenu.addEventListener('click', show);
...
+function closeModal(modal) {
+  if (modal == null) return;
+  modal.classList.remove('active');
+  overlay.classList.remove('active');
+}
+
+openModalButtons.forEach((button) => {
+  button.addEventListener('click', () => {
+    const modal = document.querySelector(button.dataset.modalTarget);
+    // eslint-disable-next-line no-use-before-define
```
Comment (prolajumokeoni) [inline index.js]: - [ ] Kindly avoid disabling linters in your project as this is bad practice, always fix linter errors and ensure all checks are green 👍🏿

**Pull request merged**